environment.py

```diff
@@ -16,3 +17,16 @@ def setup(self):
                         range(self.config.regions)]
 
         # TODO: setup agents
+        # setup containers
```

Comment:
This can be made more generic with a helper method, which abstracts this for both containers and producers.